apps/site/assets/ts/hooks/useMobileDetect.ts

```diff
@@ -0,0 +1,10 @@
+import MobileDetect from "mobile-detect";
+
+const mobileDetect = new MobileDetect(window.navigator.userAgent);
+
+// This is to primarily to be used when generting android or ios specific behavior
+// For feature detection a different library/approach should be used
+const isIPhone = (): boolean => mobileDetect.is("iPhone");
+const isAndroid = (): boolean => mobileDetect.is("Android");
+
+export { isIPhone, isAndroid };
```

Comment:
This is more of a helper function than a React hook; can we put this in `ts/helpers`?